rfcs/0114-nixos-code-of-conduct.md

```diff
@@ -0,0 +1,112 @@
+which provides a great compromise between explicit behaviors and subjective goals.
+Only minor changes have been made.
+
+- We are committed to providing a friendly, safe and welcoming environment for
+all, regardless of level of experience, gender identity and expression,
+sexual orientation, disability, personal appearance, body size, race, ethnicity, age,
+religion (or lack thereof), socioeconomic status, nationality, or other similar characteristics.
+- Please avoid using overtly sexual aliases or other nicknames that might
+detract from a friendly, safe and welcoming environment for all.
+- Please be kind and courteous. There’s no need to be mean or rude.
+- Respect that people have differences of opinion and that every design or
+implementation choice carries a trade-off and numerous costs. There is seldom a single right answer.
+- Please keep unstructured critique to a minimum. If you have solid ideas
+you want to experiment with, make a fork and see how it works.
+- We will exclude you from interaction if you insult, demean or harass anyone.
+That is not welcome behavior. We interpret the term “harassment” as including the definition in the
+[Citizen Code of Conduct](https://github.com/stumpsyn/policies/blob/b8ee50e7a9fc79c39c936830901f8863d493b7cd/citizen_code_of_conduct.md);
+if you have any lack of clarity about what might be included in that concept,
+please read their definition. In particular, we don’t tolerate behavior that excludes
+people in socially marginalized groups.
+- Private harassment is also unacceptable. No matter who you are, if you feel
+you have been or are being harassed or made uncomfortable by a community member,
+please contact one of the channel ops or any of the NixOS moderation team immediately.
+Whether you’re a regular contributor or a newcomer, we care about making this community
+a safe place for you and we’ve got your back.
+- Likewise any spamming, trolling, flaming, baiting or other attention-stealing behavior is not welcome.
+
+# Examples and Interactions
+[examples-and-interactions]: #examples-and-interactions
+
+There have been a few spectacular failures of communication and good faith within the community.
+Most notable recent examples include [ad-hominem attacks in a PR review](https://github.com/NixOS/nixpkgs/pull/120729#discussion_r621885348)
+and associated [block evasion discourse thread](https://discourse.nixos.org/t/github-block-evasion-is-not-acceptable/12763),
+heated discussions in [RFC #98](https://github.com/NixOS/rfcs/pull/98) and [RFC #111](https://github.com/NixOS/rfcs/pull/111).
+Each of these incidents have caused significant discussion on IRC (when it was still official),
+matrix, discourse, and would bleed over into github and even non-official platforms.
+These discussions are generally very polarizing, and causes an enormous amount
+of emotional and mental stress to those involved.
+These incidents are also very embarrassing for the greater nix community,
```

Comment:
```suggestion
These incidents are also very embarrassing for the entire Nix community,
```

Not sure whether it makes sense to implicitly distinguish sub-sets of the Nix community, especially since they were not even referred to in this text.